### DISCUSSION TITLE
malware detected in file default_exe.exe

### DISCUSSION THREAD

**Discussion by udi-mosh** [Security-Onion-Solutions/securityonion#8960] [General] [answered]
hi,

in versions 170 and 180 my AV found this:
"...which detected the Trojan named Artemis!EE468A4B1F55 while scanning D:\Tmp\securityonion-2.3.170-20220922.iso\REGISTRY.TAR;1\data\data\default_exe.exe".

is this o.k ?
Udi

**Comment by dougburks** [comment, marked as answer]
From https://docs.securityonion.net/en/2.3/download.html:
> If you download our ISO image and then scan it with antivirus software, it is possible that one or more of the files included in the ISO image may generate false positives. If you look at the antivirus scan details, it will most likely tell you that it alerted on a file in SecurityOnion\agrules\strelka\yara\. This is part of [Strelka](https://docs.securityonion.net/en/2.3/strelka.html#strelka) and it is being incorrectly flagged as a backdoor when it is really just a Yara ruleset that looks for backdoors. **In some cases, the alert may be for a sample EXE that is included in [Strelka](https://docs.securityonion.net/en/2.3/strelka.html#strelka) but again a false positive.**
